src/utils/update.rs

```diff
@@ -233,7 +233,13 @@ fn update_nagger_impl() -> Result<(), Error> {
                 check.latest_version()
             )).yellow()
         );
-        println_stderr!("{}", style("run sentry-cli update to update").dim());
+        if is_homebrew_install() {
+            println_stderr!("{}", style("run brew upgrade sentry-cli to update").dim());
+        } else if is_npm_install() {
+            println_stderr!("{}", style("Please use npm/yarn to update sentry-cli").dim())
```

Comment:
Currently this line will never actually run because of https://github.com/getsentry/sentry-cli/pull/306/files#diff-5f099b4f4eb2e9d89a0a2ed0f393f9c2R267. Either this branch should be removed, or the `is_npm_install` noop below.